src/main/java/Deadline.java

```diff
@@ -0,0 +1,27 @@
+public class Deadline extends Task{
+
+    private static final String LINE_SEPARATOR = "____________________________________________________________";
+    private static final String ADDED_MESSAGE = "Got it. I've added this task: \n";
```

Comment:
clearly followed coding standards by using ALL caps for constants throughout the code.